### PR TITLE
Fix Inconsistency in the seq2seq documentation.

### PR DESCRIPTION
**Pull request by Conchylicultor**
With TF 1.0, the parameters `softmax_loss_function` has changed from `Function (inputs-batch, labels-batch)` to `Function (labels-batch, inputs-batch)`.

The documentation has been updated for `sequence_loss_by_example`:
https://github.com/tensorflow/tensorflow/blob/a83290927bc2c7d64560d442d8aaec2d1cf155ab/tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py#L1063

But not for `sequence_loss` and `model_with_buckets` even if those are calling `sequence_loss_by_example` internally:
https://github.com/tensorflow/tensorflow/blob/a83290927bc2c7d64560d442d8aaec2d1cf155ab/tensorflow/contrib/legacy_seq2seq/python/ops/seq2seq.py#L1114

Edit: Here is the commit that made the change but forgot to update the documentation in some places: https://github.com/tensorflow/tensorflow/commit/24246a1097048883bc83951cda347c447c562bcc

### CODE REVIEW

**Comment by Conchylicultor** [comment]
I signed it!

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Review by lukaszkaiser — APPROVED** [review]
Thanks for spotting this and correcting.

**Comment by lukaszkaiser** [comment]
@tensorflow-jenkins test this please

**Comment by caisq** [comment]
@Conchylicultor PR Merged. Thank you for the contribution!

**Comment by Conchylicultor** [comment]
I don't know if this is voluntary but the new `seq2seq` module still use the old parameters order.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/seq2seq/python/ops/loss.py#L51

**Comment by ebrevdo** [comment]
Good question for lukasz.

**Comment by lukaszkaiser** [comment]
I think Alex wrote the new functions before the change happend. Probably we want to update them?
